File: backend/models/TheThanhVien.js
const { DataTypes } = require("sequelize");
const sequelize = require("../config/db");

const TheThanhVien = sequelize.define(
  "TheThanhVien",
  {
    id: { type: DataTypes.BIGINT, primaryKey: true, autoIncrement: true },
    ten_hang: { type: DataTypes.STRING(50), allowNull: false },
    muc_chi_tieu_tu: { type: DataTypes.DECIMAL(15, 2), allowNull: false },
    muc_chi_tieu_den: { type: DataTypes.DECIMAL(15, 2), allowNull: true },
    ty_le_giam_gia: { type: DataTypes.DECIMAL(5, 2), allowNull: false },
    diem_thuong_them: { type: DataTypes.INTEGER, allowNull: false },
    mau_the: { type: DataTypes.STRING(20), allowNull: true },
    mo_ta_quyen_loi: { type: DataTypes.TEXT, allowNull: true },
  },
  {
    tableName: "TheThanhVien",
    timestamps: false,
  },
);

module.exports = TheThanhVien;

File: backend-csharp/Models/TheThanhVien.cs
using System;
using System.Collections.Generic;

namespace WebFashion.Api.Models;

public partial class TheThanhVien
{
    public long Id { get; set; }

    public string TenHang { get; set; } = null!;

    public decimal MucChiTieuTu { get; set; }

    public decimal? MucChiTieuDen { get; set; }

    public decimal TyLeGiamGia { get; set; }

    public int DiemThuongThem { get; set; }

    public string? MauThe { get; set; }

    public string? MoTaQuyenLoi { get; set; }

    public virtual ICollection<TaiKhoan> TaiKhoans { get; set; } = new List<TaiKhoan>();
}
